src/lib/fauna-account-client.mjs

```diff
@@ -204,12 +207,50 @@ export class FaunaAccountClient {
       method: "POST",
       path: "/databases/keys",
       body: JSON.stringify({
-        path,
         role,
+        path: this.standardizeRegion(path),
         ttl: ttl || new Date(Date.now() + TTL_DEFAULT_MS).toISOString(),
         name: "System generated shell key",
       }),
       secret: this.accountKeys.key,
     });
   }
+
+  /**
+   * Transforms database paths to standardize region group naming conventions expected by
+   * the account API.
+   *
+   * @param {string} databasePath - The database path to standardize
+   * @returns {string} The standardized path
+   * @example
+   * // Returns "us-std/my-database"
+   * standardizeRegion("us/my-database")
+   *
+   * // Returns "eu-std/my-database"
+   * standardizeRegion("eu/my-database")
+   *
+   * // Returns "global/my-database"
+   * standardizeRegion("classic/my-database")
+   *
+   * @throws {TypeError} If databasePath is provided but not a string
+   */
+  standardizeRegion(databasePath) {
+    if (!databasePath) return databasePath;
+    if (typeof databasePath !== "string") {
+      throw new TypeError("Database path must be a string");
+    }
+
+    const parts = databasePath.split("/");
+    const region = parts[0].toLowerCase();
+    const rest = parts.slice(1).join("/");
+
+    const regionMap = {
+      us: "us-std",
+      eu: "eu-std",
+      classic: "global",
+    };
+
+    const standardRegion = regionMap[region] || region;
+    return `${standardRegion}/${rest}`;
```

Comment:
can you add a test (general-cli.mjs maybe?) that confirms the regions get mapped?

Reply:
Added!